lib/spdy-transport/connection.js

```diff
@@ -281,7 +281,7 @@ Connection.prototype._handleFrame = function _handleFrame(frame) {
 
   // Session window update
   if (frame.type === 'WINDOW_UPDATE' && frame.id === 0) {
-    if (state.version < 3.1 && state.autoSpdy31) {
```

Comment:
Sorry for delay! Not really.

If `autoSpdy31` is `false` - the update from 3 to 3.1 should not happen at all. That's why we have this option in place. Actually, this option makes sense only with `plain: true`, because otherwise protocol version is negotiated via NPN.